integrations/docker/images/vscode/chip-build-vscode/Dockerfile

```diff
@@ -130,7 +140,6 @@ ENV NXP_K32W0_SDK_ROOT=/opt/k32w/core
 ENV NXP_K32W1_SDK_ROOT=/opt/k32w/k32w1
 ENV NXP_SDK_ROOT=/opt/nxp-sdk/rw61x
 ENV OPENOCD_PATH=/opt/openocd/
-ENV PW_ENVIRONMENT_ROOT=/home/vscode/pigweed/env
 ENV QEMU_ESP32=/opt/espressif/qemu/qemu-system-xtensa
```

Comment:
This means that vscode images would default to `.environment` which is not desirable if run from another image (e.g. I use podman and not containers from vscode).